### PR TITLE
feat: [GTM-814]: add ci tracking events

### PR DESCRIPTION
**Pull request by nanaxuHarness**
##### Summary:

this is to add more page and track events to ci:
1) add page event to every ci page
2) add track events to ci: https://harness.atlassian.net/wiki/spaces/CING/pages/21023129820/Trial+signup+Instrumentation+WIP

##### Jira Links:

https://harness.atlassian.net/browse/PLG-814

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by nanaxuHarness** [comment]
retrigger titlecheck

**Comment by nanaxuHarness** [comment]
retrigger coverage

File: src/modules/10-common/pages/pageContext/PageName.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+export enum PAGENAME {
```
Comment (tan-nhu) [inline src/modules/10-common/pages/pageContext/PageName.ts:8]: PAGE_NAME?

**Review by tan-nhu — APPROVED** [review]

File: src/modules/10-common/pages/pageContext/PageName.ts (hunk 1 above)
Comment (nanaxuHarness) [reply src/modules/10-common/pages/pageContext/PageName.ts]: updated

**Review by mtran7 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**